### DISCUSSION TITLE
The recent update is not active, everyone is waiting for the new and old versions

### DISCUSSION THREAD

**Discussion by Jengro777** [toeverything/AFFiNE#533] [General]
The number of people interacting is less, because everyone can't find it and don't know how to discuss affine.

**Comment by Yipei-Operation** [comment]
Sorry for the late reply, for real-time communication, can join our live chat channel:
* Discord: https://discord.gg/Arn7TqJBvG
* Telegram: https://t.me/affineworkos
For other social media, can refer to our following links:
* Reddit: https://www.reddit.com/r/Affine/
* Twitter: https://twitter.com/AffineOfficial
* Medium：https://medium.com/@affineworkos
* Community：https://community.affine.pro/home
* Github: https://github.com/toeverything/AFFiNE
